src/components/termsandconditions/_termsandconditions.scss

```diff
@@ -0,0 +1,140 @@
+/* ===============Terms And Conditions ================== */
+
+.cmp-adaptiveform-termsandcondition {
+  color: $dark-gray;
+  font-size: $font-md;
+  margin: $field-margin;
+}
+
+
+.cmp-adaptiveform-termsandcondition__label-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.cmp-adaptiveform-termsandcondition__label {
+  flex: 1;
+  font-size: $font-lg;
+}
+
+.cmp-adaptiveform-termsandcondition__content-container {
+  margin-top: 10px
+}
+
+.cmp-adaptiveform-termsandcondition__content-container--modal {
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: $gray;
+}
+
+.cmp-adaptiveform-termsandcondition__content-container--modal .cmp-adaptiveform-termsandcondition__body {
+  background-color: $white;
+  margin: auto;
+  width: 600px;
+  border-radius: 4px;
+}
+
+.cmp-adaptiveform-termsandcondition__content-container--modal .cmp-adaptiveform-termsandcondition__header {
```

Comment:
Add selector like this
.cmp-adaptiveform-termsandcondition__content-container--modal{
.cmp-adaptiveform-termsandcondition__header{
}
.cmp-adaptiveform-termsandcondition__header h3{
}
cmp-adaptiveform-termsandcondition__close-button{
}
}